index/src/executors/bridge_executors/phala_to_ethereum.rs

```diff
@@ -6,15 +6,22 @@ use alloc::{
     vec::Vec,
 };
 use scale::Decode;
+use sp_runtime::{traits::ConstU32, BoundedSlice};
 use subrpc::{create_transaction, send_transaction, ExtraParam};
-use xcm::v1::{AssetId, Fungibility, Junction, Junctions, MultiAsset, MultiLocation};
+use xcm::v3::{AssetId, Fungibility, Junction, Junctions, MultiAsset, MultiLocation, Weight};
 
 #[derive(Clone)]
 pub struct ChainBridgePhala2Ethereum {
     evm_chainid: u8,
     rpc: String,
 }
 
+#[derive(Clone)]
+pub struct SygmaBridgePhala2Ethereum {
+    evm_chainid: u8,
```

Comment:
Rename suggestion: in Sygma, DomainID is used to identify the chain. We can rename to `evm_domainid`